privaterelay/settings.py

```diff
@@ -99,14 +99,12 @@
 AWS_SNS_TOPIC = config('AWS_SNS_TOPIC', None)
 AWS_SNS_KEY_CACHE = config('AWS_SNS_KEY_CACHE', 'default')
 AWS_SES_CONFIGSET = config('AWS_SES_CONFIGSET', None)
-SOCKETLABS_SERVER_ID = config('SOCKETLABS_SERVER_ID', 0, cast=int)
-SOCKETLABS_API_KEY = config('SOCKETLABS_API_KEY', None)
-SOCKETLABS_SECRET_KEY = config('SOCKETLABS_SECRET_KEY', None)
-SOCKETLABS_VALIDATION_KEY = config('SOCKETLABS_VALIDATION_KEY', None)
```

Comment:
Did you mean to nuke the socketlabs configs? Seems unrelated to a recruitment banner.

Reply:
Sorry, yes I should have commented here. The `SOCKETLABS` stuff should be removed ASAP. I could split into 2 commits maybe?

Reply:
Or, do a separate PR for socket labs and I'll blindly R+ and merge immediately, and then we won't be blocked on banner UI stuff, would also work.

Reply:
I'll make it a separate PR/change.

Reply:
Updated.